framework/encode/d3d12_capture_manager.cpp

```diff
@@ -364,6 +364,12 @@ void D3D12CaptureManager::InitializeID3D12DeviceInfo(IUnknown* adapter, void** d
     }
 }
 
+bool D3D12CaptureManager::IsAccelerationStructureResouce(format::HandleId id)
```

Comment:
Typo: IsAccelerationStructureResou**r**ce

Missing the `r`

Also, do you really need the assert? Is the state tracker ever going to be null? Check other parts of the capture manager.

Reply:
Will update.

Reply:
Done.